app/bundles/LeadBundle/Model/LeadModel.php

```diff
@@ -1920,6 +1919,10 @@ public function import($fields, $data, $owner = null, $list = null, $tags = null
             if ($company !== null) {
                 $this->companyModel->addLeadToCompany($company, $lead);
             }
+
+            if ($eventLog) {
+                $lead->addEventLog($eventLog);
```

Comment:
Can we be sure that this change will be saved to database? No `flush` is called here.

Reply:
Yep. Because of https://github.com/mautic/mautic/pull/5804/files#diff-9784b7fe0e4e02b9768b1efa4c50fe25R393 and https://github.com/mautic/mautic/pull/5804/files#diff-9784b7fe0e4e02b9768b1efa4c50fe25R396.

I also tested it and it stores all rows to the log table. Before the last commit it stored only errors.